bash/ostree

```diff
@@ -849,6 +849,7 @@ _ostree_pull_local() {
         --gpg-verify-summary
         --require-static-deltas
         --untrusted
+        --disable-verify-bindings
```

Comment:
Just wondering, did you consider making this a `--verify-bindings=<true|false>`, defaulting to `true`?
I think that would be slightly more aligned with `ostree fsck` current option, although I'm not use if the usecase is the same.

Reply:
I hadn't considered that since it didn't match any of the other options there. It's exactly the same use case as fsck. Our repos also fail that 😁

In the case of fsck it's off by default. For pulling it's on by default. I don't believe the fsck option does that kind of handling, but maybe I missed it.

Reply:
My personal taste would be to turn this PR so that at the end of the day you can do `ostree pull --verify-bindings=false`, but I'd be happy to hear somebody else's opinion on the UX here.

Reply:
AFAIK, `--flag=<true|false>` isn't really easy with glib. There's no built-in support for it, so you'd have to e.g. implement your own callback or parse the string yourself. @dbnicholson is right that it doesn't match any other options we have right now. Probably better to just stick with the same paradigm.

Reply:
Yep, I agree that is a better UX, but improving the CLI in this aspect should probably be a distinct effort.